src/graphics/graphics_device.js

```diff
@@ -829,6 +894,9 @@ pc.extend(pc, function () {
             if (texture.autoMipmap && pc.math.powerOfTwo(texture._width) && pc.math.powerOfTwo(texture._height) && texture._levels.length === 1 && !texture._compressed) {
                 gl.generateMipmap(texture._glTarget);
             }
+
+            var stats = pc.Application._currentApplication.stats.vram;
```

Comment:
Speed is not any "touched" by namespacing. This would be unmeasurable nano-optimisation.